src/uproot/source/fsspec.py

```diff
@@ -92,6 +92,9 @@ def chunk(self, start: int, stop: int) -> uproot.source.chunk.Chunk:
         Request a byte range of data from the file as a
         :doc:`uproot.source.chunk.Chunk`.
         """
+        if self.closed:
+            raise OSError(f"memmap is closed for file {self._file_path}")
```

Comment:
I also like to use `repr` when presenting strings in error messages, since it exposes when there's a weird character in the string that might be causing the problem (probably not in this case).